.github/workflows/Deploy.yaml

```diff
@@ -24,21 +30,28 @@ defaults:
 jobs:
   Deploy:
     runs-on: [ ubuntu-latest ]
+    environment: Production
```

Comment:
Doesn't this change the game?

What if _Production_ environment does not exist? Do we need more docs?

Reply:
I'll test this but no I don't believe you need the environment as long as you have an ORGPAT secret defined

Reply:
Tested it this morning. GitHub is still able to run the workflow as long as you have the OrgPat defined as a secret. As far as I can tell, the only change is that GitHub will automatically create a "Production" environment for you if it doesn't exist.